### ISSUE TITLE
When switching indicators, the limits in X and Y dialog are not updated 

### ISSUE THREAD

**Issue by angieskazka** [vizabi/vizabi#2407]
open bubble chart
check options --> zoom X and Y. the Y axis limits are 19 to 86 for life expectancy, this is correct
change axis Y to Central bank discount rate
check options --> zoom X and Y. the Y axis limits are still 19 to 86 (incorrect now)

![image](https://cloud.githubusercontent.com/assets/3648190/22939514/41a4fd84-f2df-11e6-8ef5-d1be54ce80c7.png)

**Comment by nicu-chiciuc** [comment]
This happens to all the data. Doesn't influence the functionality but seems like a big problem.

**Comment by nicu-chiciuc** [comment]
I found the problem. In the commit https://github.com/vizabi/vizabi/commit/3bdf788a74f633a5e61482dcf07799e332393b81, by setting the `dontFeedToState` variable to a string. This in turn will make the `go()` function in `bubblechart-panzoom.js` to not set the `zoomedMin`, `zoomedMax` of `axis_x` and `axis_y` when changing the indicator type. They are used in [minmaxinputs.js#L135](https://github.com/vizabi/vizabi/blob/develop/src/components/minmaxinputs/minmaxinputs.js#L135) and are set to `null` after changing the indicator.

A simple solution would be to revert the first change in the commit that I linked. Yet I'm not sure if there was a specific reason to implement this functionality.
